File: Unidad 2-Estructuras de Control/Ejercicio7.c
/* Escribir el programa para un programa que reciba un numero positivo, si este numero es mayor a 1000 se le sumara un 5%, si el numero es mayor a 3000 se le sumara otro 10% adicional y si el numero es mayor a 5000 se le sumara otro 5% adicional. */

#include <stdio.h>

/* GLOBAL VARIABLES */

int number;

/* Functions */

void entrada() {
  scanf("%d", &number);
}

void proceso(int num) {
  int result = num;
  if(num > 1000) {
    result += result * .05;
  }
  if(num > 3000) {
    result += result * .1;
  }
  if(num >5000) {
    result += result *.05;
  }
  number = result;
}

void salida() {
  printf("%d", number);
}

/* MAIN */

int main() {

  entrada();
  proceso(number);
  salida();

  return 0;
}

File: Unidad 2-Estructuras de Control/ejercicio7.py
# Ejercicio 7
# Escribir el programa para un programa que reciba un numero positivo, si este numero es mayor a 1000 se le sumara un 5%, si el numero es mayor a 3000 se le sumara otro 10% adicional y si el numero es mayor a 5000 se le sumara otro 5% adicional.
# Autor: Jorge Reyes (Equipo 'about:blank')

# Dato de entrada: valor numérico
# Dato de salida: valor numérico con el agregado correspondiente

def percentageCalc(num):
    # Se asigna el valor de entrada a la variable "result"
    result = num

    # Compara y re-asigna valores
    if (num > 1000):
        result += num * .05
    if (num > 3000):
        result += num * .1
    if (num > 5000):
        result += num * .05

    return int(result)

inp = int(input())
print(percentageCalc(inp))
